### PR TITLE
TT-193

### PR DESCRIPTION
**Pull request by MontgomeryWatts**
Allow Committee Heads and (Super)Admins to view private charges

### CODE REVIEW

File: app/charges/controllers.py [hunk 1]
```diff
@@ -69,8 +69,9 @@ def get_charges(user, user_data, broadcast = False):
     membership = committee.members.filter_by(member= user).first()
 
     for charge in charges:
-        if charge.private and membership is None:
-            continue;
+        if charge.private:
+            if (membership is None or membership.role != Roles.CommitteeHead) or not user.is_admin or not user.is_super:
```
Comment (MontgomeryWatts) [inline app/charges/controllers.py]: This could be broken down into two conditionals if we want better readability.

Comment (lukasyelle) [reply app/charges/controllers.py]: Yeah, sorry I just am seeing this now, I think that is the way to go. Always best to have the next guy in mind, the easier to understand the better.

**Comment by lukasyelle** [comment]
@MontgomeryWatts Looks like a test needs to be updated and then we'll be good to go.

**Comment by MontgomeryWatts** [comment]
I'm going to be doing similar work for this ticket: https://ritservices.atlassian.net/browse/TT-139
I'm thinking about closing/declining this PR because the work here will be duplicated in order to set a convention.

**Pull request closed without merging**